main.go

```diff
@@ -5,8 +5,13 @@ import (
 )
```

Comment:
Remove the comment that doesn't add any value and is redundant. Comments should provide context or explain non-obvious code behavior.